tests/test_cli.py

```diff
@@ -1022,11 +1022,15 @@ def test_noQC_citations_not_written_verbose(self, caplog):
         )
 
 
-@pytest.mark.parametrize(
-    "species_id", [species.id for species in stdpopsim.all_species()]
-)
+@pytest.mark.parametrize("species_id", [s.id for s in stdpopsim.all_species()])
 def test_species_simulation(species_id):
-    cmd = f"-q {species_id} -L 1 --seed 1234 10"
+    # L must be longer than any gene conversion tract lengths
+    L = 1000
+    species = stdpopsim.get_species(species_id)
+    for chrom in species.genome.chromosomes:
+        if chrom.gene_conversion_length is not None:
```

Comment:
Since the GC length for StrAga is 120Kb, I had to make this change or run this test with L > 120000 for all species, which seemed like it might take a while? I'll put an explanatory comment.